.github/templates/issue_delete_clusters_failed.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We are not using this flow in the team, please use Slack notifications and send to the `team-distribution-alerts` channel.